core/src/main/java/com/linecorp/armeria/internal/common/metric/RequestMetricSupport.java

```diff
@@ -161,6 +159,21 @@ private static void updateMetrics(
         }
     }
 
+    private static void updateRetryingClientMetrics(
+            RequestContext ctx, RequestLog log,
+            ClientRequestMetrics metrics, int childrenSize,
+            SuccessFunction successFunction) {
+
+        metrics.actualRequests().increment(childrenSize);
+
+        final boolean success = successFunction.isSuccess(ctx, log);
```

Comment:
`successFunction.isSuccess(ctx, log)` is also called in `updateMetrics` method.
Let's call it once and reuse it.